src/qt/configurenamedialog.cpp

```diff
@@ -61,7 +64,17 @@ void ConfigureNameDialog::accept()
     returnData = ui->dataEdit->text();
     returnTransferTo = ui->transferTo->text();
 
-    QDialog::accept();
+    if(!IsValidJSON(data)){
+        QMessageBox::StandardButton MessageBoxInvalidJSON = 
+        QMessageBox::question(this, "Invalid JSON",
```

Comment:
Per [this Microsoft UI guidance](https://learn.microsoft.com/en-us/windows/win32/uxguide/vis-std-icons), I think a warning icon should probably be used rather than a question icon. It's possible I'm misreading the guidance, let me know if you disagree.

Reply:
Makes sense, it is a warning after all

Reply:
I think this is not resolved yet? Or am I misreading GitHub's UI?